.gitignore

```diff
@@ -74,4 +74,3 @@ _build
 *.ptx
 *.ll
 *.bc
-*.yml
```

Comment:
What's the reason for removing this? Asking because action recording could produce `yml` files (which IMO should be ignored).

Reply:
`.travis.yml`, `appveyor.yml`, etc. There're too many files involving `yml`. For example, I added `.github/workflows/some_new_workflow.yml`, and find it's not added because of `.gitignore`.
FYI I use `.gitignore_localrc` for ignoring these outputs.

Reply:
Ah I see